src/etc/inc/plugins.inc.d/dpinger.inc

```diff
@@ -428,7 +430,6 @@ function dpinger_status()
             $status[$gwname]['status'] = $r['status'];
         }
     }
-
```

Comment:
I enjoy the solitude of return statements ;)

Reply:
woops :)